### ISSUE TITLE
Spawning in a wall

### ISSUE THREAD

**Issue by pinkpanther8732** [ScerIO/PocketMine-MP-Android#149]
Soooo. I'm new to this so I may be big dumb lol but here goes..

I created the Pocketmine server mcpe android samsung galaxy s8.

At first when I went on server to check it out, everything was fine. Decided I would try and make a faction for my friends and I so I started researching. Downloaded a couple things and it said on the console no errors so I assumed everything was fine. Went on after stopping and restarting server annnnd...

Nada lol

Tried a couple more times and nope.
So I then took everything off. Decided I would try again after a break. I also uploaded one of my resource packs with it. Console didnt show any errors. 

Went on my server and now everytime I spawn I am in a wall and suffocate. So I deleted everything again that was new on the file of my pocketmine and went back on and nothing has changed.  I continue to die in a wall lol

Palllleeease help lol

**Comment by Galvinized83** [comment]
I can confirm this happens on my fresh install too.

**Comment by Galvinized83** [comment]
@pinkpanther8732 
Stop the server, then delete the contents of the "worlds" folder. Keep the original folder, just delete the contents, then restart the server. 
Try rejoining the game - that seems to have worked for me.

**Comment by pinkpanther8732** [comment]
Thank you for the tip. I will try that today. When I 1st got the server it's from server.pro and it seems to be the only one that works I've tried other ones and I couldn't do a lot with it or even get on. The 1st time I go on it was fine for me and then a took me about 30 minutes to get my middle child on and now after I did that I was the 1 spawning in a wall lol I don't know what I did to fix it and then my oldest child is now spawning and just dying and so lol I sit here and laugh because I'm just not very smart when it comes to technology like I'm learning and this kind of stuff has started to really interests me and I just appreciate any type of help that I'm getting. I keep trying to put plug INS on the server to and I'm struggling so hard to get them to be on their. Anyway I am getting a computer soon so hopefully that will help I don't know lol bye I will definitely a look and see if that works and I'll let you know if it did thank you again.
